contracts/lupi.sol

```diff
@@ -15,17 +20,35 @@ contract lupi is owned {
     State state = State.Betting;
 
     Ticket[] tickets;
-    uint revealedCount;
+    uint revealedCount; // TODO: shall we replace this with a getter returning revealedBets.length?
 
     // bet => tickets
     mapping(uint => uint[]) revealedTickets;
     uint[] revealedBets;
 
     uint winningTicket;
 
-    function lupi() {
+    // TODO: function getBets(address _player) constant returns bets[]
```

Comment:
getBets( address player) is needed for FE. I guess we need a new mapping for it but would worth to think over the whole data structure if we can optimize it